src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,6 +69,16 @@ public RobotContainer() {
             () -> (-modifyAxis(will.getLeftY()) * Drive.MAX_VELOCITY_METERS_PER_SECOND),
             () -> (-modifyAxis(will.getLeftX()) * Drive.MAX_VELOCITY_METERS_PER_SECOND)));
 
+    // armSubsystem.setDefaultCommand(
+    //     new FunctionalCommand(
+    //         () -> {},
+    //         () -> {
+    //           armSubsystem.setPercentOutput(ControllerUtil.deadband(nick.getLeftY(), .2));
+    //         },
+    //         (interupted) -> {},
+    //         () -> false,
+    //         armSubsystem));
+
```

Comment:
Was this intentionally left in as an example?

Reply:
It was left in for ease of uncommenting. I've copied it from source control so many times I think it's better to just leave it.
Thoughts?